src/strict/processors/openai_processor.py

```
@@ -64,3 +71,27 @@ async def process(self, request: ProcessingRequest) -> OutputSchema:
+        try:
+            stream = await self.client.chat.completions.create(
+                model=self.model,
+                messages=messages,  # type: ignore
+                stream=True,
+                timeout=request.timeout_seconds,
+            )
+            async for chunk in stream:
+                if chunk.choices[0].delta.content:
+                    yield chunk.choices[0].delta.content
+        except APIError as e:
+            yield f"OpenAI streaming error: {str(e)}"
```

Ruff (0.14.14)

[warning] Use explicit conversion flag

Replace with conversion flag

(RUF010)